whipper/program/cdparanoia.py

```diff
@@ -196,7 +196,7 @@ def getTrackQuality(self):
 
         # don't go over a 100%; we know cdparanoia reads each frame at least
         # twice
-        return min(frames * 2.0 / reads, 1.0)
+        return min(frames * 2.0 / reads, 1.0) if reads else 0
```

Comment:
Actually I thought about this and wanted to preserve this beauty of code with only one return path (and the min()). If you ask me the most pythonic is to ask for forgiveness and not permission. So what about a try-expect block that catches the division by zero exception?